### ISSUE TITLE
pybind array assumed to be contiguous all the time

### ISSUE THREAD

**Issue by martinfalk** [inviwo/inviwo#674]
When data is copied from a numpy array, it is always assumed that the data is contiguous. However, this is not necessarily the case for array views! In that case, one needs to create a deep copy before using this data with Inviwo, i.e. `numpy.array(data, copy=True)`

```python
a = numpy.array([[1,2], [3,4], [5,6]])
b = a.flip(0)  # returns a view, cannot be used for a inviwo Layer / Volume
c = numpy.array(b, copy=True)   # can be used with inviwo
```
see https://docs.scipy.org/doc/numpy-1.14.2/reference/generated/numpy.flip.html

Plain copy for example here:
https://github.com/inviwo/inviwo/blob/7042a1a85de45fbfdb1f104e371f87de1232b977/modules/python3/src/pybindutils.cpp#L96-L102

**Comment by martinfalk** [comment]
Forgot to mention: when one tries to copy data from a view, Inviwo crashes due to a read of invalid memory.

**Comment by martinfalk** [comment]
Addressed in 8015b397ca878738515dca5fbf08f0e71d567c89.

https://github.com/inviwo/inviwo/blob/ae3f7869cdb19adcf5365f3bf379157125348ba0/modules/python3/src/pybindutils.cpp#L132-L164
